README.md

```diff
@@ -101,37 +127,24 @@ $ cp deployments/cni-conf.json /etc/cni/net.d/
 $ kubectl create -f deployments/crdnetwork.yaml
 ```
 
-### Build SRIOV CNI
+#### Network Object CRDs
 
-1. Compile SRIOV-CNI (dev/k8s-deviceid-model branch):
-```
-$ git clone https://github.com/intel/sriov-cni.git
-$ cd sriov-cni
-$ make
-$ cp build/sriov /opt/cni/bin
-```
-2. Create the SRIOV Network CRD
+Multus uses Custom Resource Definitions(CRDs) for defining additional network attachements. These network attachment CRDs follow the standards defined by K8s Network Plumbing Working Group(NPWG). Please refer to [Multus documentation](https://github.com/intel/multus-cni/blob/master/README.md) for more information.
+1. Create the SRIOV Network CRD
 ```
 $ kubectl create -f deployments/sriov-crd.yaml
 ```
 
-### Build and run SRIOV network device plugin
+#### DANM
```

Comment:
It might be good to have an example as Multus in Readme file on how to build the meta plugin and create sriov network attachment. This may require creating templates in the deployment folder.

Reply:
Those are already available in the DANM repository, so I chose simply link them, rather than duplicate them here.
I generally don't like duplicating things, because one of them will ultimately fall out of synch.
do you see any issue with simply linking the appropriate artifacts?

Reply:
+1
I think link to the DANM is good for its own documentations and sample deployment.

Reply:
I may have different thought on this, if we list DANM as an alternative meta plugin of Multus, then we should treat them equally, this may include:
1) provide simple example that prove to be functional in device plugin doc
2) add new tests where fits for using device plugin with DANM, this may require adding new tests in CI (ongoing work)
3) evaluate other parts of doc to make sure all the description to meta plugin also applies to DANM, otherwise, we need to add note or change the doc.

Above may not be needed if the goal is just to point user for another meta plugin solution and we don't guarantee it always works. If that's the case, we perhaps also want to add a note for that and set the right expectation for users.

Reply:
I don't consider myself as a part of the community, so the decision is yours! I'm okay either way :)
couple questions to your points:
1: examples meaning manifests? or proof from a real installation that it works?
2: Unit Tests, or functional tests? because from unit test perspective I don't think the meta plugin has any interface to the DP.
if you mean functional tests, and already have an automated K8s + Multus + DP installation sure, we can work on creating one such setup with DANM too.
I think it might take more time though, so maybe we could approach this iteratively. Or maybe use the CNCF Testbed project for this?
3: if you see other parts which need some face lifting feel free to point out! I went through and I think I updated whatever needed updating, but if you see anything else I will change that too

I will try and join the next netowrking / resource management scrum call on Monday, we could discuss the details and additional action items online!

Reply:
Ok, I went through the entire document flow again. I can see why @zshi-redhat is asking about the example deployments. 

With this change, the "Install a compatible CNI meta plugin" section doesn't tell user to pick ONE meta plugin - Multus or DANM, not both. Users who are new to this may as well proceed with both steps.  ¯\\\_(ツ)\_/¯

Worse even, the "Example deployments" section doesn't mention that it is based on the assumption that K8s is setup with Multus(at least this was the case before we introduced DANM here). Users who may want to try this out with DANM may think that, this example deployments should work with either meta plugin.

Maybe we should try to make it clearer to the users that the example deployments in this document assume that Multus is configured. To deploy with DANM they should look into relevant section provided by DANM instead adding entire DANM deployments on this doc.

Reply:
Sure! I will make the first point more explicit, and maybe fork the example output section too and add some outputs with DANM (thought most of the printouts will be similar)

Reply:
done, please re-check. the example manifests are still live under the DANM repository, but the README here now contains very detailed step-by-step examples how one can deploy them

also added example test results to showcase it works :)